test/performance/odp_ipsecfwd.c

```diff
@@ -329,6 +345,9 @@ static void print_usage(void)
 	       "  -I, --num_input_qs  Input queue count. 1 by default.\n"
 	       "  -S, --num_sa_qs     SA queue count. 1 by default.\n"
 	       "  -O, --num_output_qs Output queue count. 1 by default.\n"
+	       "  -d, --direct_rx     Use direct RX. Interfaces will be polled by workers\n"
+	       "                      directly. \"--mode\" and \"--num_input_qs\" options are\n"
+	       "                      ignored, input queue count will match worker count.\n"
```

Comment:
Now it would be quite straightforward to support sync ipsec also with scheduled I/O, but I suppose that could be a topic for another patch set. Or if you want to add it now, I would suggest adding sync mode first and then the direct mode I/O on top if it in a separate patch.

Reply:
Leaving this as future work.